fetch.js

```diff
@@ -426,7 +435,11 @@
         xhr.setRequestHeader(name, value)
       })
 
-      xhr.send(typeof request._bodyInit === 'undefined' ? null : request._bodyInit)
+      if (request._bodyArrayBuffer) {
+        xhr.send(request._bodyArrayBuffer)
```

Comment:
The tests already exists, or are you suggesting that I test for a deprecation warning? I don't know how to do that.